README.md

```diff
@@ -1,17 +1,20 @@
 # rokka/twig
 
-A twig extension to easily use [rokka](https://rokka.io) related methods in your twig templates.
+A twig extension to use [rokka](https://rokka.io) related methods in your twig templates.
 
-Docs are coming, very much work in progress, see the source or open an issue, if questions.
-
-Stable release coming soon.
+This allows you to upload images to rokka while rendering a template. As this will slow down rendering the template
+considerably, this is mainly useful with static site generators that render the pre-render the templates before
```

Comment:
Only on the first hit, now? Once an image is uploaded and we have stored the hash somewhere (by default in that .json) it shouldn't significantly slow down the rendering (apart from reading that json file)

Reply:
i rephrased to be clearer. also made clear that the twig functions also work with rokka hashes, so the extension as such definitely makes sense in all scenarios.